pages/tutorials/3.0/graphics/sprite.md

```diff
@@ -19,55 +20,73 @@ Ok, that was short but if you really don't understand what sprites and textures
 
 ## Loading a texture
 
-Before creating any sprite, we need a valid texture. The class that encapsulates textures in SFML is, surprisingly, [`sf::Texture`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Texture.php "sf::Texture documentation"). Since the only role of a texture is to be loaded and mapped to graphical entities, almost all its functions are about loading and updating it.
+Before creating any sprite, we need a valid texture.
+The class that encapsulates textures in SFML is, surprisingly, [`sf::Texture`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Texture.php "sf::Texture documentation").
+Since the only role of a texture is to be loaded and mapped to graphical entities, almost all its functions are about loading and updating it.
 
-The most common way of loading a texture is from an image file on disk, which is done with the `loadFromFile` function.
+The most common way of loading a texture is from an image file on disk, which is done with either the `loadFromFile` function or the corresponding constructor.
 
 ```cpp
-sf::Texture texture;
-if (!texture.loadFromFile("image.png"))
-{
-    // error...
-}
+sf::Texture texture("image.png");
```

Comment:
```suggestion
sf::Texture texture("image.png"); // Throws sf::Exception if an error occurs

// OR

sf::Texture texture;
if (!texture.loadFromFile("image.png"))
{
    // error...
}
```

Reply:
Found a few other places where that same comment belongs